src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -259,96 +287,110 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
                     closeButtonColor="info"
-                    onClose={this.closeNativeMagnifierModal} />
+                    onClose={this.closeNativeMagnifierModal}
+                />
 
-                <Alert show={this.state.showInvalidRegionWarning}
+                <Alert
+                    show={this.state.showInvalidRegionWarning}
                     title={strings.editorPage.messages.enforceTaggedRegions.title}
                     // tslint:disable-next-line:max-line-length
                     message={strings.editorPage.messages.enforceTaggedRegions.description}
                     closeButtonColor="info"
-                    onClose={() => this.setState({ showInvalidRegionWarning: false })} />
+                    onClose={() => this.setState({ showInvalidRegionWarning: false })}
+                />
             </div>
         );
     }
 
     private closeNativeMagnifierModal = () => {
         this.setState({
-            magnifierModalIsOpen: false,
+            magnifierModalIsOpen: false
         });
-    }
+    };
 
     private showNativeMagnifierModal = () => {
         this.setState({
-            magnifierModalIsOpen: true,
+            magnifierModalIsOpen: true
         });
-    }
+    };
 
     private onPageClick = () => {
         this.setState({
-            selectedRegions: [],
+            selectedRegions: []
         });
-    }
+    };
 
     /**
      * Called when the asset side bar is resized
      * @param newWidth The new sidebar width
      */
     private onSideBarResize = (newWidth: number) => {
-        this.setState({
-            thumbnailSize: {
-                width: newWidth,
-                height: newWidth / (4 / 3),
+        this.setState(
+            {
+                thumbnailSize: {
+                    width: newWidth,
+                    height: newWidth / (4 / 3)
+                }
             },
-        }, () => this.canvas.current.forceResize());
-    }
+            () => this.canvas.current.forceResize()
+        );
+    };
 
     /**
      * Called when the asset sidebar has been completed
      */
     private onSideBarResizeComplete = () => {
         const appSettings = {
             ...this.props.appSettings,
-            thumbnailSize: this.state.thumbnailSize,
+            thumbnailSize: this.state.thumbnailSize
         };
 
         this.props.applicationActions.saveAppSettings(appSettings);
-    }
+    };
 
     /**
      * Called when a tag from footer is clicked
      * @param tag Tag clicked
      */
     private onTagClicked = (tag: ITag): void => {
-        this.setState({
-            selectedTag: tag.name,
-            lockedTags: [],
-        }, () => this.canvas.current.applyTag(tag.name));
-    }
+        this.setState(
+            {
+                selectedTag: tag.name,
+                lockedTags: []
+            },
+            () => this.canvas.current.applyTag(tag.name)
+        );
+    };
```

Comment:
@bklukaczewski In most of the time husky is working great, but in this case I think that previous version was better :thinking:

Reply:
@johannamelly What do you think?

Reply:
Hard to say what you're referring to ;) 
Are there any issues with pre-commit? Is formatting broken?

Reply:
@bklukaczewski  No, everything is fine I was reffering to the `setState`. The formating is changing sometimes, but sometimes not. But that's not something that's really important :p 

```
        this.setState({
            selectedTag: tag.name,
            lockedTags: CanvasHelpers.toggleTag(locked, tag.name),
        }, () => this.canvas.current.applyTag(tag.name));


        this.setState(
            {
                selectedTag: tag.name,
                lockedTags: CanvasHelpers.toggleTag(locked, tag.name)
            },
            () => this.canvas.current.applyTag(tag.name)
        );
   
```